### ISSUE TITLE
Show agent health on cocoon dashboard

### ISSUE THREAD

**Issue by jmagman** [flutter/flutter#48430]
Show individual health of agents on new dashboard.  Old dashboard showed them in red at the top.

![Screen Shot 2020-01-08 at 10 31 33 AM](https://user-images.githubusercontent.com/682784/72005287-1a40d000-3202-11ea-9b76-380d5c763b73.png)

**Comment by jmagman** [comment]
\cc @CaseyHillers

**Comment by CaseyHillers** [comment]
When https://github.com/flutter/cocoon/pull/594 lands, this should be pretty easy.

**Comment by jmagman** [comment]
> When [flutter/cocoon#594](https://github.com/flutter/cocoon/pull/594) lands, this should be pretty easy.
> 
> Edit: It adds a agent tile card that makes it easy to show that information. Could be shown as a pop up on the agent in the task overlay.

Can we avoid removing the old build dashboard until it's done?

**Comment by CaseyHillers** [comment]
Thanks for the feedback on this, I updated the Agent Dashboard PR to help with showing the agent status from the build dashboard. In the task overlay, the agent has been moved to it's own button that will redirect to the agent dashboard to show the details for that agent.

![agent_redirect](https://user-images.githubusercontent.com/2148558/72291999-65ddea00-3605-11ea-85fa-ff0e3e52f7bf.png)

![agent_filter](https://user-images.githubusercontent.com/2148558/72292251-e270c880-3605-11ea-960e-4a142481197f.png)

Going back in the browser will pop back to the build dashboard.

Since this task information is already shown in an overlay, I don't know if we want to create another overlay to show the agent on the same page. Let me know if you have any ideas.
